cluster/examples/kubernetes/ceph/dashboard-external-https-openshift.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rook-ceph-mgr-dashboard-external-https
```

Comment:
How about naming this file `dashboard-loadbalancer-https.yaml` and the name field `rook-ceph-mgr-dashboard-loadbalancer`? I don't think it's specific to openshift.

Reply:
I agree. Done!